teamcity/resource_build_trigger_schedule.go

```diff
@@ -27,12 +28,12 @@ func resourceBuildTriggerSchedule() *schema.Resource {
 				Type:         schema.TypeString,
 				ForceNew:     true,
 				Required:     true,
-				ValidateFunc: validation.StringInSlice([]string{"daily", "weekly"}, false),
+				ValidateFunc: validation.StringInSlice([]string{"daily", "weekly", "cron"}, false),
```

Comment:
Is it possible to enforce the presence of `cron_schedule` (and perhaps, absence of the hour/minute/... fields) if `schedule = "cron"`?

Reply:
Each field has support for a ValidateFunc (more documentation [here](https://github.com/yext/terraform-provider-teamcity/blob/yext/vendor/github.com/hashicorp/terraform/helper/schema/schema.go#L243)). However, it doesn't look like it has support for dependencies on another field, or for fields of type schema.TypeList. I'll add logic to have the apply stage error out if `schedule = "cron"` is specified, and `cron_schedule` is not defined.

Reply:
SGTM!